components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -470,14 +474,11 @@ func hasBackendTypeChanged(currentBackendStatus, desiredBackendStatus eventingv1
 	return currentBackendStatus.Backend != desiredBackendStatus.Backend
 }
 
-func getDefaultBackendStatus() eventingv1alpha1.EventingBackendStatus {
-	return eventingv1alpha1.EventingBackendStatus{
-		SubscriptionControllerReady: utils.BoolPtr(false),
-		PublisherProxyReady:         utils.BoolPtr(false),
-		EventingReady:               utils.BoolPtr(false),
-		BEBSecretName:               "",
-		BEBSecretNamespace:          "",
-	}
+func getDesiredBackendStatus() eventingv1alpha1.EventingBackendStatus {
+	defaultStatus := eventingv1alpha1.EventingBackendStatus{}
+	defaultStatus.InitializeBackendConditions()
+	defaultStatus.EventingReady = utils.BoolPtr(true)
```

Comment:
Why EventingReady is true by default? Should it be false by default?

Also (nit) the name `getDefaultBackendStatus()` seems better to me. `getDesiredBackendStatus` seems to me like it should return the final status with everything ready and all conditions ready (like a desire :p).

Reply:
It's true since our desiredStatus would be to have eventingReady as `true`.

Actually that's the idea I propose: To desire the required status (where eventingReady is true), and if something doesn't match or breaks, update the status (to false) accordingly.